content/outcomes/cloud-native-development/specific__messaging.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Messaging
+weight: 170
+layout: single
+team:
+ - VMware Tanzu Labs
+---
+
+Event-driven applications are a natural fit for microservices as they reduce the coupling between services even further and allow for the whole system to move from central orchestration to distributed asynchronous choreography. 
```

Comment:
Is there a good article to link that explains central orchestration vs. distributed asynchronous choreography?

Reply:
A dedicated article is planned on the topic in the learning path for distributed systems.